bin/spring

```diff
@@ -4,12 +4,12 @@
 # It gets overwritten when you run the `spring binstub` command.
 
 unless defined?(Spring)
-  require "rubygems"
-  require "bundler"
+  require 'rubygems'
```

Comment:
@dogweather JSYK, we are using double quotes for everything. Can you update your linter?